.buildkite/pipeline.yml

```diff
@@ -45,6 +43,18 @@ steps:
    - nix-build tests/tezos-binaries.nix --no-out-link --arg path-to-binaries ./docker
    branches: "!master"
 
+ - label: deb source packages via docker
+   commands:
+   - nix run -f. pkgs.docker -c ./docker/docker-ubuntu-packages.sh source
+   artifact_paths:
+     - ./out/*
+ - label: deb binary packages via docker
+   commands:
+   - nix run -f. pkgs.docker -c ./docker/docker-ubuntu-packages.sh binary
```

Comment:
AFAIU most of  `docker/package/package_generator.py` is done regardless of binary/source. Don't we do something twice here?

Reply:
that's true, the only difference is `dpkg-buildpackage` flag

Reply:
however, the process of fetching and archiving sources is quite fast